src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,49 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = null
+    
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      if (printTimeCallback) {
+        printTimeCallback();
+      }
+      // printTimeCallback && printTimeCallback()
+      this.currentTime++;
+    },10)
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor((this.currentTime/100) / 60)
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor((this.currentTime/100) % 60);
   }
 
   getCentiseconds() {
-    // ... your code goes here
+    return (this.currentTime % 100)
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    return value.toString().padStart(2,'0')
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    let minutes = this.computeTwoDigitNumber(this.getMinutes());
+    let seconds = this.computeTwoDigitNumber(this.getSeconds());
+    let centiSeconds = this.computeTwoDigitNumber(this.getCentiseconds());
```

Comment:
Salvo que vayais a reasignar los valores, siempre siempre siempre const!

```suggestion
    const minutes = this.computeTwoDigitNumber(this.getMinutes());
    const seconds = this.computeTwoDigitNumber(this.getSeconds());
    const centiSeconds = this.computeTwoDigitNumber(this.getCentiseconds());
```